planet/System.c4g/LanguageDE.txt

```diff
@@ -768,7 +768,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Nutzungsbedingungen
 IDS_MSG_CLEARTHEMESSAGEBOARD=Alle Nachrichten löschen.
 IDS_MSG_CLIENT=Client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Countdown läuft nicht!
-IDS_MSG_CMD_HOSTONLY=Kein Host? Versagt!
+IDS_MSG_CMD_HOSTONLY=Nur der Host kann das Spiel starten!
```

Comment:
What's wrong with the old message?

Reply:
IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something more kind like "Nur der Host kann das tun.") or be separated from the other commands.

Reply:
The old message is nonsense. The english version sounds "Host only!" Which is more precised then this. The old message has a bad slang. So another proposal:
`IDS_MSG_CMD_HOSTONLY=Das kann nur der Host machen.`